### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mvz**
* Auto-update rake to 13.0.0
* Auto-update rubocop to 0.75.0
* Auto-update rubocop-performance to 1.5.0
* Auto-update rubocop-rspec to 1.36.0
* Autocorrect new RSpec/EmptyLineAfterExample offenses
* Fix new Performance/Count offense
* Disallow version 5.1.0 of factory_bot dependency to support JRuby builds

### CODE REVIEW

**Comment by chastell** [comment]
The JRuby errors on `require 'pathname'` missing a second argument are the weirdest, I’ll restart the jobs and see whether this happens again. :scream:

**Comment by chastell** [comment]
Same. :man_shrugging:

**Comment by mvz** [comment]
It's due to this problem: https://github.com/thoughtbot/factory_bot/issues/1331

File: Gemfile [hunk 1]
```diff
@@ -7,15 +7,15 @@ ruby RUBY_VERSION
 group :development do
   gem 'aruba',               '~> 0.14.8'
   gem 'cucumber',            '~> 3.0'
-  gem 'factory_bot',         '~> 5.0'
+  gem 'factory_bot',         '~> 5.0.2'
```
Comment (chastell) [inline Gemfile]:
```suggestion
  gem 'factory_bot',         '~> 5.0', '!= 5.1.0'
```

**Comment by mvz** [comment]
Well, I a agree one would hope so, but no fix has even been committed to master.

**Comment by mvz** [comment]
I'll rebase this mess :slightly_smiling_face:

**Comment by mvz** [comment]
Ha, it looks like an RSpec update surfaced a new spec issue :man_shrugging:.

**Comment by mvz** [comment]
Hm, it's due to this: https://github.com/rspec/rspec-expectations/pull/1125

I'm unsure why such a breaking change was merged for a minor release. :confused:

**Comment by chastell** [comment]
OMG. OK, let’s merge this in before other deps follow suit. :sweat_smile:

**Pull request merged**